compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Upgrade.hs

```diff
@@ -138,6 +137,8 @@ generateTemplateInstanceModule env externPkgId
           modName <>
           " as X"
         , "import \"" <> packageName <> "\" " <> modName
+        , "import qualified DA.Internal.LF"
+        , "import qualified DA.Internal.Prelude"
```

Comment:
You will have to import these now as `Sdk.DA.Internal.Prelude` and expose the modules with this alias in `generateAndInstallInstancesPackage` in Packaging.hs.

Reply:
I dont understand what is special about the 2 imports I added. 
Many cases of `DA.Internal.`... without an `Sdk.` prefix were already present in this file.

Please feel free to make the change you think is necessary in this PR.

Reply:
Now I understand. This change is necessary after rebasing on master.